connector/amqp10/amqp10.go

```diff
@@ -10,44 +12,44 @@ import (
 	"github.com/infrawatch/apputils/config"
 	"github.com/infrawatch/apputils/logging"
 
-	"github.com/apache/qpid-proton/go/pkg/amqp"
-	"github.com/apache/qpid-proton/go/pkg/electron"
+	amqp "github.com/Azure/go-amqp"
 )
 
 const (
 	defaultSendTimeout      = 2
-	defaultListenPrefetch   = -1
+	defaultListenPrefetch   = 1
 	defaultClientName       = "localhost"
 	defaultLinkFailureLimit = 20
+	defaultMaxParalelSend   = 128
 )
 
-// AMQP10Receiver is tagged electron receiver
-type AMQP10Receiver struct {
-	Receiver electron.Receiver
-	Tags     []string
-}
-
 // AMQP10Connector is the object to be used for communication with AMQP-1.0 entity
 type AMQP10Connector struct {
-	Address          string
-	ClientName       string
-	SendTimeout      int64
-	ListenPrefetch   int64
-	LinkFailureLimit int64
-	appName          string
-	inConnection     electron.Connection
-	outConnection    electron.Connection
-	receivers        []AMQP10Receiver
-	senders          map[string]electron.Sender
-	logger           *logging.Logger
-	interrupt        chan bool
+	Address             string
+	ClientName          string
+	SendTimeout         int64
+	ListenPrefetch      int64
+	LinkFailureLimit    int64
+	MaxParalelSendLimit int64
```

Comment:
spelling: parallel

Reply:
s/Paralel/Parallel/g

Reply:
It's also in the config (around L156), not strictly only variables in the code.